core/templates/domain/admin/admin-backend-api.service.ts

```diff
@@ -140,6 +140,48 @@ export class AdminBackendApiService {
       });
     });
   }
+
+  async startNewJobAsync(jobType: string): Promise<void> {
+    return this.http.post<void>(
+      AdminPageConstants.ADMIN_HANDLER_URL, {
+        action: 'start_new_job',
+        job_type: jobType
+      }
+    ).toPromise();
```

Comment:
[Optional] Should we define 
```
private _postAdminActionAsync(action, payload) {
  return this.http.post<void>(
      AdminPageConstants.ADMIN_HANDLER_URL, { action, ...payload }).toPromise();    
} 
```
and use _postAdminActionAsync in all the other places?

[Ignore this if you find it complicated, It's an optional comment. :)]

Reply:
I'll try it will decrease code redundancy.

Reply:
done!